### DISCUSSION TITLE
看到代码中包含了推送相关文件，但是我在助手中却没找到...

### DISCUSSION THREAD

**Discussion by ZzzHe2333** [moesnow/March7thAssistant#200] [Q&A]
![W0H9{F9ONM88 Y}4R%4_Y1Z](https://github.com/moesnow/March7thAssistant/assets/110341553/8e850005-eb23-473b-835b-6498f2b8764a)
# 想知道这一块是在哪里触发的，如何结束的时候触发。
# 想自己新增的话，该怎么新增。

**Comment by moesnow** [comment]
https://moesnow.github.io/March7thAssistant/#/assets/docs/Tutorial?id=%e6%8e%a8%e9%80%81

**Reply by ZzzHe2333** [reply]
# 看起来不知道是什么原因发送失败了0.0
